### ISSUE TITLE
Accessibility - Cannot TAB to 'Delete Folder' confirmation window and NVDA cannot reach and read it.

### ISSUE THREAD

**Issue by SergeantConfused** [bitwarden/web#948]
## Describe the Bug
When deleting a folder from the My Vault page, the 'Delete Folder' confirmation window is not in focus and you cannot TAB to it, and the NVDA screen reader cannot read the message or the buttons.

## Steps To Reproduce
1. Log into your web vault.
2. Create a test folder from the My Vault view.
3. Try to delete the test folder
4. Try to TAB to reach the buttons or message so the screen reader would read them. 

## Expected Result
To be able to reach the new window using TAB.

## Actual Result
You cannot TAB to the new 'Delete Folder' confirmation window.

## Screenshots or Videos
N/A.

## Environment
- Operating system: Windows 10 Pro.
- Browser: Firefox 88.0.
- Build Version: 2.19.0.
- NVDA: 2020.4.

## Additional Context
After opening the 'Delete Folder' confirmation window, pressing the Esc key closes the 'EDIT FOLDER' window in the background.

**Comment by maxymapp** [comment]
@SergeantConfused 'Delete Folder' confirmation window was recently removed from the flow completely. The open issue is rendered inapplicable anymore. However, I suspect similar cases should still exist when a popup is rendered over another popup. Swal library modals may need to be configured to support keyboard or replaced altogether for such scenarios.

**Comment by SergeantConfused** [comment]
Hi @maxymapp,

I just tested this again and the 'Delete Folder' confirmation window was present as shown below:

![image](https://user-images.githubusercontent.com/73389600/117791173-b6457600-b252-11eb-9925-df7d6afad1aa.png)

Can you please explain to me what you meant?


Thank you in advance,
